### ISSUE TITLE
[Doc] Document directory rectification [en metrics-tracing]

### ISSUE THREAD

**Issue by Alonexc** [apache/incubator-eventmesh#1372]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Documentation Related

Move the documents under directory https://github.com/apache/incubator-eventmesh/tree/master/docs/en/metrics-tracing 
to directory https://github.com/apache/incubator-eventmesh/tree/master/docs/en/design-document ,and delete the metrics-tracing directory.
notice:The names of documents moved to design document should be sorted by number.
eg:
<img width="292" alt="image" src="https://user-images.githubusercontent.com/91315508/191908425-79fdecef-4f9d-4590-94ee-27cc35b38282.png">

Then the next file should start with 11-.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by eshikarya** [comment]
Hi! I'm new to Open Source contribution, and would like to get started with this.
Please let me know if I can work on this?
(I can see 2 _category_.json files in each folder, is there some specific format to be followed to rename them? )

**Comment by Alonexc** [comment]
You only need to move the document. The json file in the metrics tracing directory does not need to be moved to the design document directory.

**Comment by eshikarya** [comment]
The description requires us to delete the metrics-tracing directory, won't that delete the json file present inside it as well?

**Comment by eshikarya** [comment]
@Alonexc Got it! Please assign this to me.

Also pls clarify, based on the description, we need to delete the metrics-tracing directory, which will delete the json file present inside it. Is that alright?

**Comment by qqeasonchen** [comment]
@eshikarya sure ok.

**Comment by eshikarya** [comment]
@qqeasonchen based on our previous conversation, I tried to clone and do the requested changes.
I found out that there were certain files which are automatically staged for commit as soon as I clone the project into my local repo, I stashed the files, and then tried to do the required changes to commit them. This brought down the files changed from 68 to 13. I'm not sure why or how this is happening considering I'm not handling any other files except the 2 that need to be moved.

I further tried to the same changes through GitHub to see if I can bring down files changed, it was 3 as we expect it to be but the tests are failing for this, so I'm a little unsure about how to proceed with this.
